doozerlib/brew.py

```diff
@@ -45,11 +45,11 @@ def get_watch_task_info_copy():
         return dict(watch_task_info)
 
 
-def watch_task(brew_hub, log_f, task_id, terminate_event):
+def watch_task(session, log_f, task_id, terminate_event):
```

Comment:
I applaud this change but you'd need to update all the callers:
```
🐚 grep -rIe '\bwatch_task\b' doozer*
doozerlib/olm/bundle.py:        error = brew.watch_task(
doozerlib/operator_metadata.py:        return brew.watch_task(
doozerlib/distgit.py:from .brew import watch_task, get_build_objects
doozerlib/distgit.py:            error = watch_task(self.runtime.group_config.urls.brewhub, self.logger.info, task_id, terminate_event)
```

Reply:
Oops, forgot to do that